src/main/kotlin/com/ritense/iko/mvc/controller/AggregatedDataProfileController.kt

```diff
@@ -485,6 +490,23 @@ class AggregatedDataProfileController(
         return list(query = "", pageable = Pageable.ofSize(PAGE_DEFAULT), isHxRequest = isHxRequest)
     }
 
+    @DeleteMapping("/aggregated-data-profiles/{id}/cache")
+    fun clearAdpCache(
+        @PathVariable id: UUID,
+        httpServletResponse: HttpServletResponse,
+    ): ModelAndView {
+        val aggregatedDataProfile = aggregatedDataProfileRepository.getReferenceById(id)
+        val cacheKey = aggregatedDataProfile.toCacheable().cacheKey
+        val key = cacheService.hashString(cacheKey)
+        cacheService.evict(key)
+        // TODO for some reason the loading of details is not workting nice. InnerHtml stuff. Goal is to relad the details
```

Comment:
@nielserkamp-ritense you know what I do wrong? I just want to refresh tab general.